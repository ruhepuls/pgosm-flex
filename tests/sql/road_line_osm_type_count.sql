SELECT osm_type COLLATE "C", COUNT(*)
    FROM osm.road_line
    GROUP BY osm_type COLLATE "C"
    ORDER BY osm_type COLLATE "C"
;
