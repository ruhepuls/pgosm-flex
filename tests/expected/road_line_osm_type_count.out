construction|7
cycleway|259
elevator|1
footway|7383
living_street|24
motorway|476
motorway_link|401
path|551
pedestrian|47
primary|2549
primary_link|264
proposed|11
residential|4978
road|1
secondary|1462
secondary_link|67
service|13856
services|1
steps|371
tertiary|1483
tertiary_link|63
track|67
trunk|84
trunk_link|31
unclassified|208
